internal/services/phone/callhandling/call_handling_business_hours_resource.go

```diff
@@ -0,0 +1,900 @@
+	VoiceMailGreetingID                     types.String                                            `tfsdk:"voicemail_greeting_id"`
+	WrapUpTime                              types.Int32                                             `tfsdk:"wrap_up_time"`
+}
+
+type businessHoursResourceModelCallHandlingCallDistribution struct {
+	HandleMultipleCalls          types.Bool   `tfsdk:"handle_multiple_calls"`
+	RingDuration                 types.Int32  `tfsdk:"ring_duration"`
+	RingMode                     types.String `tfsdk:"ring_mode"`
+	SkipOfflineDevicePhoneNumber types.Bool   `tfsdk:"skip_offline_device_phone_number"`
+}
+
+type businessHoursResourceModelCallForwarding struct {
+	RequirePress1BeforeConnecting types.Bool                                         `tfsdk:"require_press_1_before_connecting"`
+	EnableZoomMobileApps          types.Bool                                         `tfsdk:"enable_zoom_mobile_apps"`
+	EnableZoomDesktopApps         types.Bool                                         `tfsdk:"enable_zoom_desktop_apps"`
+	EnableZoomPhoneApplianceApps  types.Bool                                         `tfsdk:"enable_zoom_phone_appliance_apps"`
+	Settings                      []*businessHoursResourceModelCallForwardingSetting `tfsdk:"settings"`
+}
+
+type businessHoursResourceModelCallForwardingSetting struct {
+	ID          types.String `tfsdk:"id"`
+	Description types.String `tfsdk:"description"`
+	Enable      types.Bool   `tfsdk:"enable"`
+	PhoneNumber types.String `tfsdk:"phone_number"`
+}
+
+func (r *tfBusinessHoursResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	var state businessHoursResourceModel
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	output, err := r.read(ctx, &state)
+	if err != nil {
+		resp.Diagnostics.AddError("Error reading phone call handling", err.Error())
+		return
+	}
+
+	diags = resp.State.Set(ctx, &output)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+func (r *tfBusinessHoursResource) read(ctx context.Context, plan *businessHoursResourceModel) (*businessHoursResourceModel, error) {
+	dto, err := r.crud.readBusinessHours(ctx, plan.ExtensionID)
+	if err != nil {
+		return nil, fmt.Errorf("error read: %v", err)
+	}
+	if dto == nil {
+		return nil, nil // already deleted
+	}
+
+	customHoursSetting := lo.Ternary(plan.CustomHours != nil && plan.CustomHours.Settings != nil, make([]*businessHoursResourceModelCustomHoursSettings, 0), nil)
+	for _, item := range dto.customHours.settings {
+		customHoursSetting = append(customHoursSetting, &businessHoursResourceModelCustomHoursSettings{
+			Weekday: item.weekday,
+			Type:    item.typ,
+			From:    lo.Ternary(item.typ.ValueInt32() == 2, item.from, types.StringNull()),
+			To:      lo.Ternary(item.typ.ValueInt32() == 2, item.to, types.StringNull()),
+		})
```

Comment:
I would like to have your comments on why it needs to be null if it is `2`.

Reply:
added https://github.com/folio-sec/terraform-provider-zoom/pull/27/commits/ef488d48c964c306fa17ff6d6d70a35ef7bd12a3.